demos/security-compare-asset-allocations/demo.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8" />
+        <link rel="stylesheet" href="demo.css" />
+        <script src="https://code.highcharts.com/dashboards/datagrid.js"></script>
+        <script src="https://code.highcharts.com/highcharts.js"></script>
```

Comment:
Please add a11y module and remove redundant scripts (eg grid). The same about CSS stylesheets

Reply:
Edit: Or add Grids :)